### DISCUSSION TITLE
ご意見フォームを復活させてほしい

### DISCUSSION THREAD

**Discussion by k1350** [increments/qiita-discussions#207] [要望・アイデア・不具合]
ご意見フォーム（discussions 運用開始以前に Qiita のフッターの『ご意見』リンクから遷移できていたフォーム）を復活させてほしいです。  
今でも過去のブログ記事からアクセスは可能ですが、Qiita からの導線が無いのでまだ使えるのかがわかりません。

どうしても公の場での言及は避けるべきだと思われる意見を送るために必要としています。  
（たとえば特定のアカウントへの批判になりかねない意見など）

公の場では避けるべき意見を問い合わせとして送る行為はかなり悪い体験だったので、できればもうしたくありません。  
ご検討いただけますと幸いです。

追記：復活させていただきたいのは https://qiita.com/feedback/new です

**Comment by tanaka-asami415** [comment]
@k1350さん
ご意見ありがとうございます
フォームへの導線が分かりにくく申し訳ありません。

Qiita Discussions 上では質問しづらい内容についてですが、
こちらのフォームをご利用ください :pray:
https://support.qiita.com/hc/ja/requests/new

こちらは
READMEの「Qiita Discussions 上では質問しづらい内容について」か

左上に固定しているdiscussion「Qiita Discussionsへようこそ」からも
アクセスいただけるようにしていたのですが

@k1350さんと同様に困られている方も少なくないのではと思い
タイトルを変更してみました🙇

引き続き何かありましたら、ご意見お待ちしております！

**Reply by k1350** [reply]
@tanaka-asami415 

起票内容がわかりづらく申し訳ありません。
そちらの URL については把握しており、起票内容ではそちらを『問い合わせ』と呼んでおりました。
復活させていただきたいのは下記です。
https://qiita.com/feedback/new

下記の Qiita Blog 記事から『ご意見』としてリンクされているページになります。
[GitHub Discussionsを正式に運用開始します](https://blog.qiita.com/qiita-discussions-general-availability/)

**Reply by k1350** [reply]
「検討しているのでしばらく待ってほしい」「回答を用意するのに時間がかかる」のようなことでいいので、今この議題がどういうステータスなのか教えていただけませんか。  
そういうやり取りが可能であるということもこの場所の意味だと思っています。

以前『問い合わせ』に意見を送ったときも沈黙されたのですが、運営の方が一生懸命検討していて沈黙しているのと、単に優先度を下げて無視・放置しているのは区別がつきません。  
無視・放置しているわけではないのなら、沈黙する前に「しばらく待って」と言ってくださると待っている側の気持ちがずいぶん違うので、可能ならお願いいたします。

**Reply by tanaka-asami415** [reply]
@k1350さん

回答が遅くなってしまい、大変申し訳ございません🙏
こちらこそ、初めにご質問内容を取り違えてしまい申し訳ないです。

[既に現在のフォーム](https://support.qiita.com/hc/ja/requests/new)をご利用いただいた上での
discussionだったのですね。大変失礼いたしました。

現フォームの「ご意見・ご要望」からの送信と
[以前の「ご意見」フォーム](https://qiita.com/feedback/new)からの送信の違いを検証してみたところ
以前の「ご意見」フォームですと、返信をしていないことが明確に記載されておりましたが

現在のフォームからですと
「内容を確認の上、弊社サポートチームが折り返しご連絡いたします。」という内容の
自動返信メールが一律で送られてしまうため、

この辺りでk1350さんのおっしゃられる、悪い体験をさせてしまったのではないかと推察いたしました。
（違っていたら、申し訳ありません）

現状、お送りくださった内容はすべて拝見していますが、
ご意見・ご要望に対しては、全てにお返事ができていない状況です。

にもかかわらず、誤解させてしまうような記載となってしまっているのは
良くないため、現フォーム等の表記を修正するようにいたします。

対応メンバーも増やし、以前より早く
ご対応できるようしていきたいと考えております。

移行期間のため、しばらくご不便をおかけするかもしれませんが
今後プライベートなお問い合わせは、現在のフォームを使用いただき
その他、ご要望・意見に関しましては、discussionsをご活用いただけると幸いです🙏

引き続き何かありましたら、ご意見お待ちしております！

**Reply by k1350** [reply]
@tanaka-asami415 

以前のご意見フォームは復活しないとのことで承知しました。

現フォーム等の表記の修正については早急に行ったほうがよろしいと存じます。

『誤解させてしまうような表記』と仰いますが、そもそも以前のご意見フォームには

> 返信の必要な内容については、こちら からお問い合わせください。

という記載で現フォームへのリンクが貼られており、そちらから送れば返信が来ることが示唆されていました。

現フォーム上でも「意見・要望には返信しないことがある」とは一言も書かれていないと思います。

更に現フォームの自動返信メールには「折り返し連絡する」と書いてあるわけです。

現フォームから意見や要望を送ったユーザーが返信を期待することが『誤解』と言われる所以はありません。  
客観的に見て、運営側が「意見や要望には返信しなくてもよい」と考えていたことのほうが誤解です。

『運営側が返信をすると言ったのに約束を守らなかった』というのが客観的事実です。

今までに意見・要望を現フォームから送信したユーザーからは相当の不信感を持たれた可能性がある、と認識を改めるほうがよろしいかと存じます。

---

あとお手数ですがクローズラベルの付与をお願いいたします🙏
自分でつけても良いと書いてあったので探したのですが、権限がないとラベルをつけられないかもしれないです……

**Reply by tanaka-asami415** [reply]
@k1350 さん

忌憚ないご意見を詳細にいただき、誠にありがとうございます。

今回、k1350さんにdiscussionをいただいたことで
現運用の問題に気づくことができました。本当にありがとうございます。

我々も単に「返信をしなくてもよい」と考えているわけではなく
全てのご意見にお返事ができていないことは、大変心苦しく感じております。

しかしながら、対応に割ける時間や人員が限られる中、どうしても限界があり、
現状、全てのご意見・ご要望に返信ができていない状況を
少しでも改善したい！と考えた方法がdiscussionsでした。

今後は、ご意見・ご要望はdiscussionsで集約していきたいと考えておりますが
k1350さんからもご指摘いただいた通り、現フォームの表記は早急に変更いたします。

今回頂いたご意見を真摯に受け止め、今後の運営改善に努めて参ります。
このたびは、誠に申し訳ございませんでした。

本件クローズさせていただきます🙏
ラベルを付与できる権限もすぐに確認いたします！
